ruleset/rules/0998-aws-security-hub-rules.xml

```diff
@@ -0,0 +1,491 @@
+  </rule>
+
+  <rule id="99840" level="12">
+    <if_sid>99811</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^DocumentDB.3$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon DocumentDB manual cluster snapshot is public.</description>
+    <group>nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99841" level="12">
+    <if_sid>99812</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^EC2.1$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - EBS snapshot is publicly restorable.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.4,pci_dss_7.2.1,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99842" level="12">
+    <if_sid>99812</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^EC2.19$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Security groups allow unrestricted access to ports with high risk.</description>
+    <group>nist_800_53_AC.4,nist_800_53_CA.9,nist_800_53_CM.2,nist_800_53_CM.7,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99843" level="12">
+    <if_sid>99818</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^EMR.2$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon EMR block public access setting not enabled.</description>
+    <group>nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99844" level="12">
+    <if_sid>99819</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^ES.2$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Elasticsearch domain is publicly accessible.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.2,pci_dss_1.3.4,pci_dss_1.3.6,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+    <mitre>
+      <id>T1530</id>
+    </mitre>
+  </rule>
+
+  <rule id="99845" level="12">
+    <if_sid>99821</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^IAM.4$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - IAM root user access key exist.</description>
```

Comment:
exists

Reply:
This has been resolved https://github.com/wazuh/wazuh/pull/23234/commits/1d4bfae8c2701e7cc0cda7b3f8df0965bdb225f8.